frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -1132,13 +1195,13 @@ mod feasibility_check {
 				.compact
 				.votes1
 				.iter_mut()
-				.filter(|(_, t)| *t == 3u16)
+				.filter(|(_, t)| *t == 3 as TargetIndex)
```

Comment:
I think using `as` is really not great since it casts values and i believe may hide errors?

Can this be `TargetIndex::from(3)` instead?

Reply:
Yup, can and will do.